DEVELOPMENT.md

```diff
@@ -136,31 +136,32 @@ Kubernetes cluster in your designated environment, if necessary.
 ### Deploy Istio
 
 ```shell
-STABLE_VERSION=$(curl https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/istio-stable)
-kubectl apply -f "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${STABLE_VERSION}/istio-crds.yaml"
-while [[ $(kubectl get crd gateways.networking.istio.io -o jsonpath='{.status.conditions[?(@.type=="Established")].status}') != 'True' ]]; do
-  echo "Waiting on Istio CRDs"; sleep 1
-done
-kubectl apply -f "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${STABLE_VERSION}/istio-minimal.yaml"
-kubectl apply -f ./third_party/net-istio.yaml
+LATEST_VERSION=$(curl https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/istio-latest)
+wget "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${LATEST_VERSION}/install-istio.sh"
+chmod +x install-istio.sh
 ```
 
-Follow the
-[instructions](https://www.knative.dev/docs/serving/gke-assigning-static-ip-address/)
-if you need to set up static IP for Ingresses in the cluster.
+If you want to install Istio without mesh, run the following command
+```shell
+wget "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${LATEST_VERSION}/istio-ci-no-mesh.yaml"
+./install-istio.sh istio-ci-no-mesh.yaml
+```
 
-If you want to adopt preinstalled Istio, please check whether the
-`cluster-local-gateway` Service is deployed in namespace `istio-system` or not
-(you can check by running
-`kubectl get service cluster-local-gateway -n istio-system`). If it's not
-installed, please install it with following command. You could also adjust
-parameters if needed.
+If you want to install Istio with mesh, run the following command
+```shell
+wget "https://raw.githubusercontent.com/knative-sandbox/net-istio/master/third_party/${LATEST_VERSION}/istio-ci-mesh.yaml"
```

Comment:
Also, bit of a nit, but seems like a shame to use curl in one place and wget elsewhere. I think curl is shipped by default on Mac, but wget needs compiling from homebrew. Could we just use curl throughout?